src/player.cpp

```diff
@@ -2017,7 +2017,7 @@ void player::memorial( std::ostream &memorial_file, std::string epitaph )
     //HP
 
     const auto limb_hp = [this, &memorial_file, &indent]( const std::string & desc,
-                                                          const body_part bp ) {
+    const body_part bp ) {
```

Comment:
I've "prettified" this line before, but astyle doesn't seem to like me putting this at the back of the line. to me it just looks weird if not outright misleading starting the linebreak right where the original expression started - at first glance it looks like `const body_part bp` is a new declaration here, not a continuation of the lambda functions parameter list.
Any suggestions how this could be made nice without going against the astyle rules?